src/utilities/codemirror/codemirror-utils.ts

```diff
@@ -120,10 +120,10 @@ export function getDefaultVariableArgs(parameters: VariableDeclaration[]): strin
         return parameter.allowable_values && parameter.allowable_values.length > 0
           ? `"${parameter.allowable_values[0]}"`
           : parameter.enum_name
-            ? `${parameter.enum_name}`
+            ? `"${parameter.enum_name}"`
             : 'UNKNOWN';
       default:
-        throw Error(`unknown argument type ${parameter.type}`);
+        return `ERROR:"${parameter.name}"`;
```

Comment:
Is it okay to no longer be throwing errors here?

Reply:
Yes, we are displaying both the error message in the editor and the parameter name that is causing the issue. When the error occurs, the autocomplete feature doesn't work and the user has to dig around and find the right argument to use for the variable.

Reply:
I added a unit test for this as well